docs/smart-contracts/getting-started/setup.mdx

```diff
@@ -95,18 +95,33 @@ Report issues and share feedback about the Stellar CLI [here](https://github.com
 
 ### Usage
 
-Run the `stellar` command and you should see output like below.
-
-```sh
-stellar
-```
+Run the `stellar --help` command and you should see output like below.
 
 ```console
-$ stellar
+$ stellar --help
 Build, deploy, & interact with contracts; set identities to sign with; configure networks; generate keys; and more.
 
 Stellar Docs: https://developers.stellar.org
-CLI Reference: https://github.com/stellar/stellar-cli/blob/main/FULL_HELP_DOCS.md
+CLI Full Hep Docs: https://github.com/stellar/stellar-cli/tree/main/FULL_HELP_DOCS.md
+
+The easiest way to get started is to generate a new identity:
+
+    stellar config identity generate alice
```

Comment:
```suggestion
    stellar keys generate alice
```

Reply:
fix here and also on stellar/stellar-cli.